packages/app-mobile/components/screens/Note.tsx

```diff
@@ -738,6 +738,7 @@ class NoteScreenComponent extends BaseScreenComponent {
 				didCancel: false,
 				error: null,
 				type: 'image/jpg',
+				name: `Photo taken at ${time.formatMsToLocal(Date.now())}`,
```

Comment:
```suggestion
				name: `${_('Photo taken at')} ${time.formatMsToLocal(Date.now())}`,
```

User facing strings need to be localized. This is done using the `_` function which is already imported at the top of this file.

Reply:
Done! Dumb question: I refreshed the translation using `yarn buildTranslations`, how do I also update the translation for mobile?

Reply:
Thanks, but actually we don't update the translations in PR as that creates too many changes, so please could you revert this? We'll update the translations in a separate commit later on.

Reply:
Reverted! Thank you.